ksql-metastore/src/main/java/io/confluent/ksql/metastore/model/StructuredDataSource.java

```diff
@@ -24,7 +24,7 @@
 import io.confluent.ksql.util.timestamp.TimestampExtractionPolicy;
 
 @Immutable
-abstract class StructuredDataSource<K> implements DataSource<K> {
+public abstract class StructuredDataSource<K> implements DataSource<K> {
```

Comment:
Having this package private is a deliberate thing.  It's an implementation detail. As such no other code should rely on it.  Just use the interface.  

```suggestion
abstract class StructuredDataSource<K> implements DataSource<K> {
```

Reply:
Yes this makes perfect sense. Still getting used to the refactor.